packages/clay-alert/src/ClayAlertBase.js

```diff
@@ -148,13 +155,14 @@ ClayAlertBase.STATE = {
 		.value(true),
 
 	/**
-	 * Flag to indicate if alert should be automatically closed.
+	 * Flag to `true` to indicate whether the alert should be closed
+	 * automatically with the default time.
 	 * @default false
 	 * @instance
 	 * @memberof ClayAlertBase
-	 * @type {?bool}
+	 * @type {?(bool|number)}
 	 */
-	autoClose: Config.bool().value(false),
+	autoClose: Config.oneOfType([Config.bool().value(false), Config.number()]),
```

Comment:
If like I suggest we configure the default times in the specific components this property should be only of type number or null.